types/src/key.rs

```diff
@@ -501,14 +550,18 @@ impl FromBytes for Key {
                 let (account_hash, rem) = AccountHash::from_bytes(remainder)?;
                 Ok((Key::Withdraw(account_hash), rem))
             }
+            tag if tag == KeyTag::Local as u8 => {
+                let (local_addr, rem) = LocalAddr::from_bytes(remainder)?;
+                Ok((Key::Local(local_addr), rem))
+            }
             _ => Err(Error::Formatting),
         }
     }
 }
 
 impl Distribution<Key> for Standard {
     fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Key {
-        match rng.gen_range(0..9) {
+        match rng.gen_range(0..=9) {
```

Comment:
Why don't we just bump to 10?

Reply:
I used inclusive range as it felt more natural to use `0..=9` when there are exactly 9 match branches